### ISSUE TITLE
Vanish Erroring on latest Paper-Version.

### ISSUE THREAD

**Issue by ImInTheICU** [TheBlackEntity/ServerSystem#14]
```
[19:44:24 WARN]: java.lang.ClassNotFoundException: net.minecraft.network.protocol.game.PacketPlayOutPlayerInfo
[19:44:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:183)
[19:44:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:150)
[19:44:24 WARN]:        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
[19:44:24 WARN]:        at java.base/java.lang.Class.forName0(Native Method)
[19:44:24 WARN]:        at java.base/java.lang.Class.forName(Unknown Source)
[19:44:24 WARN]:        at java.base/java.lang.Class.forName(Unknown Source)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.vanish.packets.VanishPacket_Reflection_Latest.setVanish(VanishPacket_Reflection_Latest.java:128)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.vanish.Vanish.setVanish(Vanish.java:77)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.commands.executable.VanishCommand.onCommand(VanishCommand.java:35)
[19:44:24 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[19:44:24 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155)
[19:44:24 WARN]:        at org.bukkit.craftbukkit.v1_19_R3.CraftServer.dispatchCommand(CraftServer.java:906)
[19:44:24 WARN]:        at org.bukkit.craftbukkit.v1_19_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64)
[19:44:24 WARN]:        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264)
[19:44:24 WARN]:        at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:322)
[19:44:24 WARN]:        at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:306)
[19:44:24 WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2281)
[19:44:24 WARN]:        at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$20(PlayerConnection.java:2241)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.b(IAsyncTaskHandler.java:59)
[19:44:24 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
[19:44:24 WARN]:        at net.minecraft.server.TickTask.run(TickTask.java:18)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1342)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.bi(MinecraftServer.java:1319)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1312)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.i_(MinecraftServer.java:1290)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1178)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
[19:44:24 WARN]:        at java.base/java.lang.Thread.run(Unknown Source)
[19:44:24 WARN]: java.lang.ClassNotFoundException: net.minecraft.network.protocol.game.PacketPlayOutPlayerInfo$PlayerInfoData
[19:44:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:183)
[19:44:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:150)
[19:44:24 WARN]:        at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.vanish.packets.VanishPacket_Reflection_Latest.setVanish(VanishPacket_Reflection_Latest.java:171)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.vanish.Vanish.setVanish(Vanish.java:77)
[19:44:24 WARN]:        at ServerSystem.jar//me.entity303.serversystem.commands.executable.VanishCommand.onCommand(VanishCommand.java:35)
[19:44:24 WARN]:        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45)
[19:44:24 WARN]:        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155)
[19:44:24 WARN]:        at org.bukkit.craftbukkit.v1_19_R3.CraftServer.dispatchCommand(CraftServer.java:906)
[19:44:24 WARN]:        at org.bukkit.craftbukkit.v1_19_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64)
[19:44:24 WARN]:        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264)
[19:44:24 WARN]:        at net.minecraft.commands.CommandDispatcher.performCommand(CommandDispatcher.java:322)
[19:44:24 WARN]:        at net.minecraft.commands.CommandDispatcher.a(CommandDispatcher.java:306)
[19:44:24 WARN]:        at net.minecraft.server.network.PlayerConnection.a(PlayerConnection.java:2281)
[19:44:24 WARN]:        at net.minecraft.server.network.PlayerConnection.lambda$handleChatCommand$20(PlayerConnection.java:2241)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.b(IAsyncTaskHandler.java:59)
[19:44:24 WARN]:        at java.base/java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source)
[19:44:24 WARN]:        at net.minecraft.server.TickTask.run(TickTask.java:18)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.d(IAsyncTaskHandler.java:153)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandlerReentrant.d(IAsyncTaskHandlerReentrant.java:24)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1342)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.x(IAsyncTaskHandler.java:126)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.bi(MinecraftServer.java:1319)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1312)
[19:44:24 WARN]:        at net.minecraft.util.thread.IAsyncTaskHandler.c(IAsyncTaskHandler.java:136)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.i_(MinecraftServer.java:1290)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1178)
[19:44:24 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320)
[19:44:24 WARN]:        at java.base/java.lang.Thread.run(Unknown Source)
[19:44:24 ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'vanish' in plugin ServerSystem v1.7.7
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:155) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_19_R3.CraftServer.dispatchCommand(CraftServer.java:906) ~[paper-1.19.4.jar:git-Paper-482]
        at org.bukkit.craftbukkit.v1_19_R3.command.BukkitCommandWrapper.run(BukkitCommandWrapper.java:64) ~[paper-1.19.4.jar:git-Paper-482]
        at com.mojang.brigadier.CommandDispatcher.execute(CommandDispatcher.java:264) ~[paper-1.19.4.jar:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:322) ~[?:?]
        at net.minecraft.commands.Commands.performCommand(Commands.java:306) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.performChatCommand(ServerGamePacketListenerImpl.java:2281) ~[?:?]
        at net.minecraft.server.network.ServerGamePacketListenerImpl.lambda$handleChatCommand$20(ServerGamePacketListenerImpl.java:2241) ~[?:?]
        at net.minecraft.util.thread.BlockableEventLoop.lambda$submitAsync$0(BlockableEventLoop.java:59) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(Unknown Source) ~[?:?]
        at net.minecraft.server.TickTask.run(TickTask.java:18) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.util.thread.BlockableEventLoop.doRunTask(BlockableEventLoop.java:153) ~[?:?]
        at net.minecraft.util.thread.ReentrantBlockableEventLoop.doRunTask(ReentrantBlockableEventLoop.java:24) ~[?:?]
        at net.minecraft.server.MinecraftServer.doRunTask(MinecraftServer.java:1342) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.server.MinecraftServer.d(MinecraftServer.java:197) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.util.thread.BlockableEventLoop.pollTask(BlockableEventLoop.java:126) ~[?:?]
        at net.minecraft.server.MinecraftServer.pollTaskInternal(MinecraftServer.java:1319) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.server.MinecraftServer.pollTask(MinecraftServer.java:1312) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.util.thread.BlockableEventLoop.managedBlock(BlockableEventLoop.java:136) ~[?:?]
        at net.minecraft.server.MinecraftServer.waitUntilNextTick(MinecraftServer.java:1290) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1178) ~[paper-1.19.4.jar:git-Paper-482]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:320) ~[paper-1.19.4.jar:git-Paper-482]
        at java.lang.Thread.run(Unknown Source) ~[?:?]
Caused by: java.lang.NullPointerException: Cannot invoke "java.lang.Class.getDeclaredConstructor(java.lang.Class[])" because "clazz" is null
        at me.entity303.serversystem.vanish.packets.VanishPacket_Reflection_Latest.setVanish(VanishPacket_Reflection_Latest.java:178) ~[ServerSystem.jar:?]
        at me.entity303.serversystem.vanish.Vanish.setVanish(Vanish.java:77) ~[ServerSystem.jar:?]
        at me.entity303.serversystem.commands.executable.VanishCommand.onCommand(VanishCommand.java:35) ~[ServerSystem.jar:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[paper-api-1.19.4-R0.1-SNAPSHOT.jar:?]
        ... 23 more

```

**Comment by TheBlackEntity** [comment]
Should be fixed in commit [89376c7e68cf8082fb900a2933c1ed3003143713](https://github.com/TheBlackEntity/ServerSystem/commit/89376c7e68cf8082fb900a2933c1ed3003143713)

**Comment by TheBlackEntity** [comment]
Please update to the latest ServerSystem version
